src/style.css

```diff
@@ -1 +1,43 @@
 /* add styles here */
+.html {
+  margin: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  padding-right: 4px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1:nth-child(-n+1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2:nth-child(-n+2) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--3:nth-child(-n+3) {
```

Comment:
The selector `.stars--3:nth-child(-n+3)` should be `.stars--3 .stars__star:nth-child(-n+3)` to correctly target the first three stars within the `.stars--3` div.